values.yaml

```diff
@@ -211,9 +211,14 @@ server:
     # replicas. If you'd like a custom value, you can specify an override here.
       maxUnavailable: null
 
-  # Definition of the serviceaccount used to run Vault.
-  serviceaccount:
-    annotations: {}
+# Definition of the serviceaccount used to run Vault.
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: true
```

Comment:
Can you make this `enabled` instead of `create` to be more consistent with other options?